contrib/vagrant/start.sh

```diff
@@ -596,6 +596,9 @@ create_k8s_config
 
 cd "${dir}/../.."
 
+# Store git version in a file so that .git does not need to be synced to the VM
+make GIT_VERSION envoy/SOURCE_VERSION
```

Comment:
What happens when using NFS and not rsync?

Reply:
GIT_VERSION file is used only if .git directory does not exist. However, I see that envoy/SOURCE_VERSION is used by the bazel scripts whenever it exists, so I'll have to hix that!

Reply:
I modified the envoy/bazel/get_workspace_status script to ignore envoy/SOURCE_VERSION if git repo is available.